src/pages/home/sidebar/SidebarScreen/index.native.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import BaseSidebarScreen from './BaseSidebarScreen';
+
+export default function SidebarScreen(props) {
```

Comment:
Add `displayName`, `propTypes` for `SidebarScreen`.

Reply:
When I add propTypes I had to add this lines so that the required fields come . But I removed it from baseSidebar.
```js
export default compose(
    withNavigation,
    withLocalize,
    withWindowDimensions,
    withOnyx({
        allPolicies: {
            key: ONYXKEYS.COLLECTION.POLICY,
        },
        betas: {
            key: ONYXKEYS.BETAS,
        },
        isCreatingWorkspace: {
            key: ONYXKEYS.IS_CREATING_WORKSPACE,
        },
    }),
)(SidebarScreen);
```